SetCurrentVersion.cmd

```diff
@@ -1,4 +1,4 @@
 set MAJOR=1
-set MINOR=5
+set MINOR=6
```

Comment:
we don't need to update version more than once between nuget updates. (and we haven't updated 1.5 to nuget). just fyi. don't need to revert this.